tmt/base.py

```diff
@@ -574,6 +547,10 @@ def export(self, *, format_: ExportFormat = ExportFormat.YAML,
             if key.startswith('_'):
                 continue
 
+            # TODO: override the method, or provide better way to filter out this class var
+            if key == '_export_plugin_registry':
+                continue
```

Comment:
Aren't all keys starting with an underscore already filtered in the condition above?

Reply:
Yes, I didn't realize that, but I'd maybe keep this explicit check where it is - it draws attention to the fact there really needs to be a better solution than ad hoc checks, sort of a pebble in my personal shoe.

Reply:
Ack, makes sense.